orchestra/frontend/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
i know we joked about not needing this, but i wonder if we need it for index.html to work?

Reply:
Hmmm, I don't remember much about our newindex.html discussion, mind jogging my memory about it? Also, this file seems to be compiled and used upon building the app, so I'm not sure if we would want to remove this.